package.json

```diff
@@ -70,10 +71,11 @@
         "terser": "^4.4.0",
         "testy": "file:./tools/npm_packages/testy",
         "tmp": "0.1.0",
-        "tsickle": "0.33.1",
+        "ts-lit-plugin": "^1.1.9",
+        "tsickle": "0.38.0",
         "tsutils": "2.27.2",
         "typeorm": "0.2.18",
-        "typescript": "3.1.6",
+        "typescript": "^3.7.5",
```

Comment:
🎉  I tried updating typescript here a week or so ago and had issues. Were there changes in rules_typescript to enable it?

Reply:
I've been using 3.7.5 since the release with no issues.

Reply:
I had to do tsickle too, and a strange jasmine typings problem needed a node types update